demo/README.md

```diff
@@ -0,0 +1,13 @@
+## beaver-logger demo
+Allows you to test your client-side and server-side changes.
+
+#### Dependency
+This setup requires spinning up a local server to host the node application. I use `http-server`.  `npm i http-server -g`.
+
+#### Setup
+1. Install node packages required for the demo in the `/demo`folder.
```

Comment:
There's a helpful `--prefix` option we can use here to prevent having to cd between directories:

```
npm install --prefix ./demo
```